backend/server/controllers/questions.js

```diff
@@ -0,0 +1,155 @@
+const Question = require('../models').Question;
+const User = require('../models').User;
+
+module.exports = {
+    create(req, res) {
+        // check that params are not null, undefined or empty string
+        if (!req.body.text) {
+            return res.status(400).send({
+                status: 400,
+                message: 'The attribute "name" of an instance of "Experiment" cannot be empty.'
```

Comment:
The field is 'text' not, 'name. Please, fix the error message. See similar errors below.